lib/koans/03_atoms.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Atoms do
+  use Koans
+
+  koan "Atoms are sort of like strings" do
+    adam = :human
+    assert adam == ___
+  end
+
+  koan "Strings can be converted to atoms, and vice versa" do
+    assert String.to_atom("atomized") == ___
+    assert Atom.to_string(:stringified) == ___
+  end
+
+  koan "Atoms are often used as keys, because they're faster than strings" do
```

Comment:
Since we can't illustrate the dot access (or lack there of) yet, maybe just remove it. The point "are faster than" isn't really illustrated anyway.